examples/3d/3d_scene.rs

```diff
@@ -100,16 +99,10 @@ fn setup(
         ..default()
     });
     // camera
-    commands.spawn((
-        Camera3dBundle {
-            transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
-            ..default()
-        },
-        DepthPrepassSettings {
-            depth_resource: true,
-            output_normals: true,
-        },
-    ));
+    commands.spawn((Camera3dBundle {
```

Comment:
I think you should be able to just remove the (,) here since it's just using the bundle alone now.